opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -518,6 +518,28 @@ message DoubleHistogramDataPoint {
   // (Optional) List of exemplars collected from
   // measurements that were used to form the data point
   repeated DoubleExemplar exemplars = 8;
+
+  // Note: the definition used to for min and max fields below is
+  // based on temporality in a way meant to match the Prometheus
+  // Summary data type used with quantiles 0 and 1, respectively, when
+  // paired with cumulative temporality.  This uses a fixed width that
+  // SHOULD be up to 5-10 minutes or for as long as data is available.
+  // https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#summary
```

Comment:
The OpenMetrics doc [link](https://github.com/OpenObservability/OpenMetrics/blame/main/specification/OpenMetrics.md#L293) says:
> Quantile values SHOULD represent the recent values. Commonly this would be over the last 5-10 minutes.

Perhaps we should adjust the wording to say something similar, for instance: "Prometheus quantiles are computed on observations more recent than a fixed max age, which is configured in clients but not known to prometheus. This interval width SHOULD be set to represent recent values. It is commonly set to 10 minutes."

I know the OpenMetrics doc says 5-10 minutes, but it defaults to 10 minutes in the prometheus client implementations:
- [java client](https://github.com/prometheus/client_java/blob/master/simpleclient/src/main/java/io/prometheus/client/Summary.java#L72-L73)
- [go client](https://github.com/prometheus/client_golang/blob/master/prometheus/summary.go#L75)
- [python client](https://github.com/prometheus/client_python/blame/9118c02a9dfee530af9e83924a88be1ac4994bc0/README.md#L150) doesn't have quantile support
- [ruby client](https://github.com/prometheus/client_ruby/blame/master/README.md#L171) doesn't have quantile support

Reply:
I changed this to say 10 minutes or as long as data is available (i.e., "up to 10 minutes").